app/views/components/_metadata.html.erb

```diff
@@ -3,12 +3,11 @@
   inline ||= false
   list_classes = %w(app-c-metadata__list)
   list_classes << 'app-c-metadata__list--inline' if inline
-  data_attributes ||= nil
```

Comment:
This also needs to be removed from the example: https://github.com/alphagov/content-publisher/blob/dbe3451b3c6ce3979b20e2a0014b7ab97cb32bc5/app/views/components/docs/metadata.yml#L8-L10

Reply:
Good spot - I've removed that now.